pkg/cni/cilium/cilium.go

```diff
@@ -218,6 +244,8 @@ func GetAppInstallOverrideValues(cluster *kubermaticv1.Cluster, overwriteRegistr
 		}
 	} else {
 		values["kubeProxyReplacement"] = "disabled"
+		values["sessionAffinity"] = true
+		valuesCni["chainingMode"] = "portmap"
```

Comment:
by default, KKP runs cilium with kube-proxy, this needs portmap plugin and session affinity enabled to pass conformance tests (otherwise, `HostPort` is not functional)
https://docs.cilium.io/en/latest/installation/cni-chaining-portmap/#portmap-hostport

Reply:
configured in https://github.com/kubermatic/kubermatic/pull/12752/commits/8ac82a3019a71e296d1e670fc9aef5b4adf47a7b